lib/response.js

```diff
@@ -520,5 +522,12 @@ module.exports = {
       message: this.message,
       header: this.header
     };
+  },
+
+  /**
+   * Flush any set headers, and begin the body
+   */
+  flushHeaders: function(){
+    this.res.write('');
```

Comment:
That error means somewhere in the code path there was a `res.statusCode = undefined` or `delete res.statusCode`.